astroid/node_classes.py

```diff
@@ -2440,6 +2440,9 @@ def bool_value(self):
         """
         return bool(self.value)
 
+    def infer(self, context=None, **kwargs):
```

Comment:
You don't need this because we already set an `_infer` method in `inference.py` for a bunch of other nodes.